doc/nrf/security/crypto/crypto_architecture.rst

```diff
@@ -160,3 +162,27 @@ When using the TF-M Crypto Service implementation, keys from the PSA Crypto API
 * Protected Storage (PS) - One of :ref:`ug_tfm_architecture_rot_services_application` that provides secure storage within the Trusted Firmware-M environment.
 
 For more information about the storage integration for the TF-M Crypto Service implementation, see :ref:`ug_psa_certified_api_overview_secstorage` and :ref:`ug_tfm_services`.
+
+.. _ug_crypto_architecture_implementation_standards_ironside:
+
+IronSide Secure Element implementation
+======================================
+
+The IronSide Secure Element (IronSide SE) implementation provides a PSA Crypto API interface access for applications running on nRF54H20.
```

Comment:
"provides a PSA Crypto API interface access"

provides a access ?

The language here is off no?

Reply:
maybe just drop "access"?

Reply:
Yeah, this sounds weird indeed.